src/main/java/com/excella/foodinc/service/FoodService.java

```diff
@@ -0,0 +1,32 @@
+package com.excella.foodinc.service;
+
+
+import com.excella.foodinc.object.Food;
+import java.util.HashMap;
+
+
+public class FoodService {

```

Comment:
I would separate the service from the dummy data. Make a separate dummy data class that holds these objects and have your service use that (for now) then when we swap to the real stuff, your service will require minimal changes